validator/rpc/intercepter_test.go

```diff
@@ -72,6 +74,9 @@ func TestServer_AuthTokenHandler(t *testing.T) {
 		require.NoError(t, err)
 		testHandler.ServeHTTP(rr, req)
 		require.Equal(t, http.StatusUnauthorized, rr.Code)
+		errJson := &httputil.DefaultJsonError{}
+		require.NoError(t, json.Unmarshal(rr.Body.Bytes(), errJson))
+		require.StringContains(t, "unauthorized", errJson.Message)
```

Comment:
If you capitalize the above, then:
```suggestion
		require.StringContains(t, "Unauthorized", errJson.Message)
```